internal/driver/glfw/menu_bar_item.go

```diff
@@ -112,11 +112,22 @@ func (i *menuBarItem) Show() {
 // Tapped toggles the activation state of the menu bar.
 // It shows the item’s menu if the bar is activated and hides it if the bar is deactivated.
 // Implements: fyne.Tappable
-func (i *menuBarItem) Tapped(*fyne.PointEvent) {
+func (i *menuBarItem) Tapped(pos *fyne.PointEvent) {
 	if i.Parent.active {
 		i.Parent.deactivate()
 	} else {
+		// Set then menu bar's index to the tapped items index and update focus
 		i.Parent.activateChild(i)
+		for j := 0; j < len(i.Parent.Items); j++ {
```

Comment:
I don't understand this loop...
`i.Parent.Items[i.Parent.index]` is `i` so the item that has just been activated is being defocussed (or "upped").
Possibly this is because I don't understand what `index` is

Reply:
I have changed the name "index" to "currentItem", as it indicates the menu item selected. There is unfortunately a mix of "focused". The focused item receiveing the keyboard data is the MenuBar, while the indicated (focus-colored) item is one of the menu items in the list. Perhaps each menu item should recieve keyboard input?

Reply:
THat might be cleaner, would probably help with code encapsulation, that said left/right may still make sense at the top level.